sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3430,6 +3430,25 @@ object GpuOverrides extends Logging {
           GpuGetJsonObject(lhs, rhs)
       }
     ),
+    expr[JsonTuple](
```

Comment:
We either need to have JsonTuple off by default or we need a way to check the "fields" to be sure that they are things that we can cover and fall back to the CPU if we cannot.

Reply:
Thank you very much! I updated JsonTuple to check for special characters in field arguments and fall back to CPU if something is detected.